### DISCUSSION TITLE
Table Block removed, New Table Design missing search fields

### DISCUSSION THREAD

**Discussion by jackflack69** [Budibase/budibase#13744] [Help] [answered]
**Checklist**
- [ ] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: Docker Single Image 
  - Budibase Version: <2.26.2> 
  - App Version: <N/A> 

**Describe the bug**
Refer to #10288 

**To Reproduce**
Make a search table through a text field.

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-8274/table-block-removed-new-table-design-missing-search-fields">BUDI-8274 Table Block removed, New Table Design missing search fields</a></p>

**Comment by aptkingston** [comment]
Hi @jackflack69. That's correct, the new table block doesn't have a setting for search fields any more. Your options are either:
- Build it yourself, which is very easy. Just add a form block above your table, then configure your table to be filtered using those form fields. This is exactly how we did it with the old table.
- Use the built in searching (hover over a column header and you'll see a search icon). This doesn't work for all types, so it depends on what you need to search.

**Reply by aptkingston** [reply, marked as answer]
Just for extra clarity - here's an example.

Here's my form block above my table. Both are using the same data source (a table called Employees) and I've added 2 fields to search - First Name and Last Name.
<img width="1385" alt="image" src="https://github.com/Budibase/budibase/assets/9075550/35d37ab3-25fb-487d-9b64-7e5d9047b67a">

Then I just configure my table to have 2 filters:
<img width="923" alt="image" src="https://github.com/Budibase/budibase/assets/9075550/2ef942c1-3d73-44ff-97d9-0ba75fa6df25">

And that's it, it will work just like the old table settings did.

**Reply by Angeeelyn** [reply]
Hello! I'm new here, and I came across the same issue with the search fields in the Budibase Table Block. Since I'm still learning, I’m having trouble understanding how it works. Could you provide a step-by-step tutorial to help me out? I’d really appreciate it—thank you!
